apps/storefront/app/grunnleggende/designelementer/farger/page.mdx

```diff
@@ -22,19 +23,130 @@ export default ({ children }) => (
+  alt='Skjermbilde av Figma som vise hvor du finner plugins'
+  boxShadow={false}
+  dataUnstyled
+/>
+
+* `Navn` Det første leddet er navnet på fargeskalaen. 
+Som standard kommer Designsystemet med fargeskalaene Success, Danger, Warning, Info og Neutral.
+Flere skalaer kan legges til ved hjelp av [Temabyggeren](https://theme.designsystemet.no/).
+
+* `Gruppe` Hver fargeskala er delt inn i gruppene Background, Surface, Border, Text og Base. 
+Disse gruppene beskriver bruksområdene til fargene. Border-gruppen skal for eksempel brukes på rammer (strokes).
+
+<Image
+  src='/img/tomat2.png'
+  alt='Skjermbilde av Figma som vise hvor du finner plugins'
+  boxShadow={false}
+  dataUnstyled
+  caption='Viser fargeskalaen til den grønne Success fargen.'
+/>
+
+
+* `Variant` Inne i hver gruppe finnes det varianter som beskriver hvordan fargene ser ut eller skal brukes. 
+Tinted betyr for eksempel at fargen har et hint av farge i seg, mens Hover betyr at fargen er tenkt brukt for en interaktiv tilstand.
+
+<Image
+  src='/img/tomat3.png'
+  alt='Skjermbilde av Figma som vise hvor du finner plugins'
+  boxShadow={false}
+  dataUnstyled
+  caption='Viser hvordan de semantiske fargene kan brukes i designet. Eksempelet bruker fire forskjellige fargeskalaer. Danger, Neutral og to forskjellige blåskalaer.'
+/>
+
+
+## Oversikt og forklaring av farger
+Hver fargeskala består av totalt 16 farger, nøye utformet for å dekke ulike behov i designet. 
+Nedenfor finner du en oversikt over de forskjellige fargene og deres tiltenkte bruksområder.
+
+
+| Navn      | Bruksområde |
+| ----------- | ----------- |
+| 1.  Background default     | Standard bakgrunnsfarge.       |
+| 2.  Background tinted      | Bakgrunn med et hint av farge i seg.        |
+| 3.  Surface default        | Standard farge for overflater / komponenter.        |
+| 4.  Surface tinted         | Overflater / komponenter med et hint av farge i seg.        |
+| 5.  Surface hover          | Hover-farge til overflater / komponenter.       |
+| 6.  Surface active         | Active-farge til overflater / komponenter.       |
+| 7.  Border subtle          | Border-farge med lav kontrast til dekorativ bruk (skillelinjer).     |
+| 8.  Border default         | Standard border-farge til skjemakomponenter og meningsbærende elementer.       |
+| 9.  Border strong          | Border-farge med høy kontrast for ekstra synlighet.      |
+| 10. Text subtle            | Tekst- og ikonfarge med lavere kontrast.        |
+| 11. Text default           | Tekst- og ikonfarge med høy kontrast og god synlighet.      |
+| 12. Base default           | Standardfarge for solide bakgrunner.       |
+| 13. Base hover             | Hover-farge for solide bakgrunner.          |
+| 14. Base active            | Active-farge for solide bakgrunner.          |
+| 15. Base contrast subtle   | Farge med god kontrast mot Base default.     |
+| 16. Base contrast default  | Farge med god kontrast mot Base default og Base hover.      |
```

Comment:
I suggest we create a simple component for this information that uses the [colorMetadata](https://github.com/digdir/designsystemet/blob/next/packages/cli/src/colors/colorMetadata.ts) its important to make sure this data is correct across multiple apps.